### PR TITLE
Implement CSP headers and other small fixes

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Implement Content Security Policy headers.

Fix bug in `InputSelect` where both value and default value were defined.

Add writing direction prop to html tag.

Enable minification using SWC.

Add more security headers based on: https://nextjs.org/docs/advanced-features/security-headers

## Justify why they are needed

We are using CSP in Web Onboarding and we should aim to at least maintain the same level of security of the new site.

I didn't copy over all the values since there are many scripts we are not going to use in this new setup.

If this config grows we should probably move it to another module.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1237** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1237" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1237?utm_source=stack-comment).

**Review by amandaflod — APPROVED** [review]
Great initiative! 🙌 Can't say I know what everything does but I trust that you know what's up 😉

File: apps/store/next.config.js [hunk 1]
```diff
@@ -12,6 +13,50 @@ const nextConfig = {
...
+  // Reduce latency when the user clicks a link
+  // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-DNS-Prefetch-Control
+  {
+    key: 'X-DNS-Prefetch-Control',
+    value: 'on',
+  },
+  // https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
+  {
+    key: 'Content-Security-Policy',
+    value: ContentSecurityPolicy.replace(/\s{2,}/g, ' ').trim(),
```
Comment (guilhermespopolin) [inline apps/store/next.config.js:52]: Is this regex replacing line breaks into empty spaces? 🤔

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/next.config.js (hunk 1 above)
Comment (robinandeer) [reply apps/store/next.config.js:52]: hmm good question -- I just copied from here: https://nextjs.org/docs/advanced-features/security-headers#content-security-policy

Comment (guilhermespopolin) [reply apps/store/next.config.js:52]: So `/s` matches any empty space. The `{2, }` part I still don't understand even with the explanation 😅 
Ah, nevermind

image

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: 30FhAldM57A3l0yLdhsW.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1237)

**Pull request merged**